subcommittees/mentoring/README.md

```diff
@@ -1,73 +1,100 @@
+- support current mentoring activities or initiate new ones
+- join the mentoring committee
+	- Sign up for the mentoring committee [mailing list][comm-mailing-list] and attend the next mentoring committee meeting (scheduled on [this etherpad](http://pad.software-carpentry.org/scf-mentoring))
 
-We welcome new members to the subcommittee.  If you are interested in
-joining our community-building efforts, please email the [mailing
-list](#mailing-list).
+For a complete summary of different mentoring roles, see [this page](roles.md).  
 
-## Steering Comittee Liaison
+## Committee Members 
 
-* Rayna Harris (@raynamharris, [@raynamharris](https://twitter.com/raynamharris))
+### Leadership
+
+* Chair
+	* Christina Koch (@ChristinaLK, [@_christinaLK](https://twitter.com/_christinaLK))
+* Vice-Chair/Steering Committee Liaison
+	* Rayna Harris (@raynamharris, [@raynamharris](https://twitter.com/raynamharris))
+* Secretary
+	* To be filled in March 2017
+* Discussion Session Coordinators
+	* To be filled in March 2017
 
-## Current Members
+### Current Members
 
 * Belinda Weaver (@weaverbel, [@cloudaus](https://twitter.com/cloudaus))
-* Bill Mills (@BillMills, [@billdoesphysics](https://twitter.com/billdoesphysics))
-* Carol Willing (@willingc, [@WillingCarol](https://twitter.com/WillingCarol))
-* Christina Koch, Chair (@ChristinaLK, [@_christinaLK](https://twitter.com/_christinaLK))
-* Daniel Chen (@chendaniely, [@chendaniely](https://twitter.com/chendaniely))
 * Karin Lagesen (@karinlag, [@karinlag](https://twitter.com/karinlag))
 * Kate Hertweck (@k8hertweck, [k8hert](https://twitter.com/k8hert))
 * Marian Schmidt (@marschmi, [@micro_marian](https://twitter.com/micro_marian))
-* Mariela Perignon (@mperignon)
-* Michael Sarahan (@msarahan)
-* Phil Rosenfield (@philrosenfield, [@philrosenfield](https://twitter.com/philrosenfield))
+* Christina Koch (@ChristinaLK, [@_christinaLK](https://twitter.com/_christinaLK))
 * Rayna Harris (@raynamharris, [@raynamharris](https://twitter.com/raynamharris))
 * Susan McClatchy (@smcclatchy, [@SueMcclatchy](https://twitter.com/SueMcclatchy))
 * Tracy Teal (@tracykteal, [@tracykteal](https://twitter.com/tracykteal))
+* Erin Becker
+* Tobin Magle
+* Mark Laufersweiler
+* Mateusz Kuzak
 
-## Past Members
+### Past Members
 
 * Sheldon McKay (@mckays630)
 * Tiffany Timbers (@ttimbers, [@TiffanyTimbers](https://twitter.com/TiffanyTimbers))
+* Phil Rosenfield (@philrosenfield, [@philrosenfield](https://twitter.com/philrosenfield))
+* Bill Mills (@BillMills, [@billdoesphysics](https://twitter.com/billdoesphysics))
+* Carol Willing (@willingc, [@WillingCarol](https://twitter.com/WillingCarol))
+* Daniel Chen (@chendaniely, [@chendaniely](https://twitter.com/chendaniely))
+* Mariela Perignon (@mperignon)
+* Michael Sarahan (@msarahan)
 
 [blog]: https://software-carpentry.org/blog/
 [blog-archives]: https://software-carpentry.org/blog/categories/#SLUG
```

Comment:
These also look like stale references:

```
$ git cat-file -p origin/pr/175:subcommittees/mentoring/README.md  | grep blog
[blog]: https://software-carpentry.org/blog/
[blog-archives]: https://software-carpentry.org/blog/categories/#SLUG
```

Does the mentoring subcommittee not intend to blog about their activity?  I'd rather fill them in (and reference the blog somewhere earlier in the README, e.g. like the [template does][1]).  But if the mentoring subcommittee does not intend to blog about their activity, then these references should be removed.

[1]: https://github.com/swcarpentry/board/blob/09b7e2264b85510cfeda5653e2a8611893614df1/subcommittees/template/README.md#outreach